rerun_py/src/catalog/catalog_client.rs

```diff
@@ -127,10 +134,17 @@ impl PyCatalogClient {
 
     //TODO(#9360): `dataset_from_url()`
 
-    fn get_table(self_: Py<Self>, id: EntryIdLike, py: Python<'_>) -> PyResult<Py<PyTable>> {
+    /// Get a table by name or id.
+    ///
```

Comment:
```suggestion
```

Reply:
Reverting this. this becomes a malformed python docstring per ruff &co.